Source/Core/Core/ConfigManager.h

```diff
@@ -95,7 +95,7 @@ struct SConfig : NonCopyable
   bool bDSPThread = false;
   bool bDSPHLE = true;
   bool bSyncGPUOnSkipIdleHack = true;
-  bool bForceNTSCJ = false;
+  bool bSelectNTSCAsJapanese = false;
```

Comment:
My opinion is that is specifically shouldn't "force". A distinctive feature of the old one was that setting it to true would force the VI bit to be true, but setting the setting to false would just make the VI bit be auto-detected. The new implementation doesn't force the VI bit in that sense. And we should especially not let this new implementation use the same INI name as the old implementation, because then setting the language to Japanese in a new version (innocuous enough, as long as language override isn't enabled) would make the NTSC-J VI bit always be on for non-NTSC-J games in older versions (rarely useful and causes unexpected problems).